modules/sc-mesh-secure-deployment/src/2_0/features/cbma/tools/utils.py

```diff
@@ -19,6 +20,78 @@
 logger = logger_instance.get_logger()
 
 
+######### MOVED FROM CLASSES #############
+
+def batman(batman_interface):
+    try:
+        subprocess.run(["ip", "link", "add", "name", batman_interface, "type", "batadv"], check=True)
+
+        batman_exec(batman_interface,"batman-adv")
```

Comment:
just to make sure: **batman_exec()** function sink's all _subprocess.CalledProcessError_ exceptions. 
It will just log some stuff but will not allow any _CalledProcessError_ leave the function )

Reply:
I removed this unneeded try-except as this function is only being called by another one. Both can be joined (and maybe some more) but there are unit tests for these functions so I'll leave them be for now until I have time to do a proper refactor :+1:

Reply:
thx a lot )